docs/tutorials/services/plan-motion-with-arm-gripper.md

```diff
@@ -74,10 +74,36 @@ if err != nil {
 {{% /tab %}}
 {{< /tabs >}}
 
-Once the Motion Service can be accessed, some familiar features become available.
-The Motion Service has a method that can get the *pose* of a component relative to a [*reference frame*](../../../services/frame-system/).
-In the tutorial where we interacted with an arm component, we used the `EndPosition` method to determine the pose of the end effector of `myArm`.
-The `GetPose` method provided by the Motion Service serves a similar function to `EndPosition`, but allows for querying of pose data with respect to other elements of the robot (such as another component or the robot's fixed "world" frame).
+Once the Motion Service is accessed, some familiar features become available.
+The Motion service has a method that can get the *pose* of a component relative to a [*reference frame*](../../../services/frame-system/).
+In the tutorial where we interacted with an arm component, we used the `GetEndPosition` method to determine the pose of the end effector of `myArm`.
```

Comment:
Do you mind adding a link to the previous tutorial here? I think that's the one referenced

Reply:
There are three links to that tutorial in the two sections before this. Would it not be overkill?